{
  "Faults": [
    {
      "latitude": 51.572821732708896,
      "type": "LV",
      "reference": "MW5966",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-25T10:30:00.000+0000",
      "affectedCustomerCount": 13,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ceaece81-9f60-41e4-9905-8a18b826598c",
      "minLongitude": -1.7518,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.750139746316053,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T09:36:00.000+0000",
      "estimatedRestoration": "2023-11-25T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW5966'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW5966",
      "maxLongitude": -1.7482,
      "minLatitude": 51.5714,
      "name": "SN3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN3 4DL",
        "SN3 4DN",
        "SN3 4DP",
        "SN3 4DR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7518,
              51.5736
            ],
            [
              -1.7482,
              51.5736
            ],
            [
              -1.7482,
              51.5714
            ],
            [
              -1.7518,
              51.5714
            ],
            [
              -1.7518,
              51.5736
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T10:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5736,
      "etrUpdate": true
    },
    {
      "latitude": 51.67165041053607,
      "type": "HV",
      "reference": "MW6484",
      "etaUpdate": true,
      "affectedCustomerCount": 74,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f10b636c-a971-4a28-b74d-5e50dc0254c1",
      "minLongitude": -1.6323,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5889975347193084,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T17:00:00.000+0000",
      "estimatedRestoration": "2023-11-26T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6484'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6484",
      "maxLongitude": -1.5404,
      "minLatitude": 51.6246,
      "name": "SN7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN7 7NL",
        "SN7 7NN",
        "SN7 7NW",
        "SN7 7SW",
        "SN7 8AQ",
        "SN7 8BG",
        "SN7 8BH",
        "SN7 8BQ",
        "SN7 8DT",
        "SN7 8DU",
        "SN7 8DX",
        "SN7 8DY",
        "SN7 8DZ",
        "SN7 8EB",
        "SN7 8ED",
        "SN7 8JR",
        "SN7 8JS",
        "SN7 8JT",
        "SN7 8JU",
        "SN7 8JX",
        "SN7 8JY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6323,
              51.7053
            ],
            [
              -1.5404,
              51.7053
            ],
            [
              -1.5404,
              51.6246
            ],
            [
              -1.6323,
              51.6246
            ],
            [
              -1.6323,
              51.7053
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T17:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7053,
      "etrUpdate": true
    },
    {
      "latitude": 51.83614428628398,
      "type": "HV",
      "reference": "MW6564",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c6a5e928-c67f-4923-9a01-598818bb33cd",
      "minLongitude": -1.749,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7451163115651762,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T19:17:00.000+0000",
      "estimatedRestoration": "2023-11-26T06:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6564'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6564",
      "maxLongitude": -1.7239,
      "minLatitude": 51.8328,
      "name": "GL54 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GL54 3DL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.749,
              51.846
            ],
            [
              -1.7239,
              51.846
            ],
            [
              -1.7239,
              51.8328
            ],
            [
              -1.749,
              51.8328
            ],
            [
              -1.749,
              51.846
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T19:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.846,
      "etrUpdate": true
    },
    {
      "latitude": 51.38566438113576,
      "type": "LV",
      "reference": "MW6562",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-25T20:30:00.000+0000",
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "115aeb5b-ad5d-4251-b2e2-6228f93223e3",
      "minLongitude": -0.9171,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9101991709372911,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T19:10:00.000+0000",
      "estimatedRestoration": "2023-11-26T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6562'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6562",
      "maxLongitude": -0.9033,
      "minLatitude": 51.3833,
      "name": "RG2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG2 9LZ",
        "RG2 9PN",
        "RG2 9PP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9171,
              51.39
            ],
            [
              -0.9033,
              51.39
            ],
            [
              -0.9033,
              51.3833
            ],
            [
              -0.9171,
              51.3833
            ],
            [
              -0.9171,
              51.39
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T20:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.39,
      "etrUpdate": true
    },
    {
      "latitude": 51.522052606522884,
      "type": "LV",
      "reference": "MW6580",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-25T21:30:00.000+0000",
      "affectedCustomerCount": 53,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "624bee67-97fe-4b05-a7c3-03239efb85a9",
      "minLongitude": -0.6157,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6136249468923884,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T00:10:00.000+0000",
      "estimatedRestoration": "2023-11-26T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6580'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6580",
      "maxLongitude": -0.6098,
      "minLatitude": 51.5183,
      "name": "SL1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL1 3AZ",
        "SL1 3BD",
        "SL1 3BL",
        "SL1 3BN",
        "SL1 3BW",
        "SL1 3EA",
        "SL1 3EB",
        "SL1 3ED",
        "SL1 3EG",
        "SL1 3EQ",
        "SL1 4XE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6157,
              51.5242
            ],
            [
              -0.6098,
              51.5242
            ],
            [
              -0.6098,
              51.5183
            ],
            [
              -0.6157,
              51.5183
            ],
            [
              -0.6157,
              51.5242
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T21:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5242,
      "etrUpdate": true
    },
    {
      "latitude": 51.37929246207068,
      "type": "LV",
      "reference": "MW6632",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-25T21:20:00.000+0000",
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8edd48a6-03ed-43f8-ab1e-e2fee290be4d",
      "minLongitude": -2.1356,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.130903224842982,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T20:44:00.000+0000",
      "estimatedRestoration": "2023-11-26T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6632'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6632",
      "maxLongitude": -2.1291,
      "minLatitude": 51.3772,
      "name": "SN12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN12 7AA",
        "SN12 7AB",
        "SN12 7AZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1356,
              51.3806
            ],
            [
              -2.1291,
              51.3806
            ],
            [
              -2.1291,
              51.3772
            ],
            [
              -2.1356,
              51.3772
            ],
            [
              -2.1356,
              51.3806
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-25T21:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3806,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-24T11:02:00.000+0000",
      "latitude": 51.456272419226934,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW5164'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW5164",
      "reference": "MW5164",
      "maxLongitude": -0.9729,
      "etaUpdate": true,
      "minLatitude": 51.4558,
      "name": "RG1 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG1 1EU",
        "RG1 2AY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9739,
              51.4566
            ],
            [
              -0.9729,
              51.4566
            ],
            [
              -0.9729,
              51.4558
            ],
            [
              -0.9739,
              51.4558
            ],
            [
              -0.9739,
              51.4566
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "5079b89d-1f6f-4253-9c04-8480273e5100",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.9739,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4566,
      "etrUpdate": true,
      "longitude": -0.9733302572038313,
      "resolved": false
    },
    {
      "latitude": 51.57017383892507,
      "type": "LV",
      "reference": "MW6707",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d8cd983e-93f3-4e58-a3a1-ee233dc76e7e",
      "minLongitude": -0.5691,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.563217501290189,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T01:22:00.000+0000",
      "estimatedRestoration": "2023-11-26T07:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6707'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6707",
      "maxLongitude": -0.5591,
      "minLatitude": 51.5667,
      "name": "SL9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL9 7HP",
        "SL9 7HS",
        "SL9 7HW",
        "SL9 7NT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5691,
              51.5719
            ],
            [
              -0.5591,
              51.5719
            ],
            [
              -0.5591,
              51.5667
            ],
            [
              -0.5691,
              51.5667
            ],
            [
              -0.5691,
              51.5719
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-26T03:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5719,
      "etrUpdate": true
    },
    {
      "latitude": 51.5118128646524,
      "type": "LV",
      "reference": "MW6576",
      "etaUpdate": true,
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "426235b0-f306-4e17-bcdb-a6c58ac5d12c",
      "minLongitude": -0.3291,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3276028988264347,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T01:28:00.000+0000",
      "estimatedRestoration": "2023-11-26T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6576'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6576",
      "maxLongitude": -0.326,
      "minLatitude": 51.51,
      "name": "W13 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "W13 0RA",
        "W13 0RE",
        "W13 0RF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3291,
              51.5125
            ],
            [
              -0.326,
              51.5125
            ],
            [
              -0.326,
              51.51
            ],
            [
              -0.3291,
              51.51
            ],
            [
              -0.3291,
              51.5125
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5125,
      "etrUpdate": true
    },
    {
      "latitude": 51.41464859604944,
      "type": "LV",
      "reference": "MW6714",
      "etaUpdate": true,
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0606492a-890d-4bbc-89fb-4cf68e02fd67",
      "minLongitude": -0.6962,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6924439822964752,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T02:07:00.000+0000",
      "estimatedRestoration": "2023-11-26T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6714'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6714",
      "maxLongitude": -0.6878,
      "minLatitude": 51.4109,
      "name": "SL5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL5 8TA",
        "SL5 8TB",
        "SL5 8TD",
        "SL5 8TE",
        "SL5 8TF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6962,
              51.4184
            ],
            [
              -0.6878,
              51.4184
            ],
            [
              -0.6878,
              51.4109
            ],
            [
              -0.6962,
              51.4109
            ],
            [
              -0.6962,
              51.4184
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4184,
      "etrUpdate": true
    },
    {
      "latitude": 51.500606811656986,
      "type": "HV",
      "reference": "MW6720",
      "etaUpdate": true,
      "affectedCustomerCount": 1005,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0a809ec4-d5b1-46c3-9efd-edf7f6316739",
      "minLongitude": -1.4002,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.320083759154891,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T02:32:00.000+0000",
      "estimatedRestoration": "2023-11-26T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6720'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6720",
      "maxLongitude": -1.2455,
      "minLatitude": 51.398,
      "name": "OX11 & RG14 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "OX11 0RP",
        "RG14 9TG",
        "RG18 9JT",
        "RG18 9JU",
        "RG18 9NU",
        "RG18 9QZ",
        "RG20 6QU",
        "RG20 7AA",
        "RG20 7AD",
        "RG20 7AE",
        "RG20 7AG",
        "RG20 7AH",
        "RG20 7AJ",
        "RG20 7AL",
        "RG20 7AN",
        "RG20 7AP",
        "RG20 7AQ",
        "RG20 7AR",
        "RG20 7AS",
        "RG20 7AT",
        "RG20 7AU",
        "RG20 7AW",
        "RG20 7AX",
        "RG20 7AY",
        "RG20 7AZ",
        "RG20 7DA",
        "RG20 7DB",
        "RG20 7DE",
        "RG20 7DF",
        "RG20 7DG",
        "RG20 7DJ",
        "RG20 7DN",
        "RG20 7HH",
        "RG20 7HN",
        "RG20 7HP",
        "RG20 7HZ",
        "RG20 7JA",
        "RG20 7JB",
        "RG20 7JD",
        "RG20 7JE",
        "RG20 7JF",
        "RG20 7JG",
        "RG20 7JH",
        "RG20 7JL",
        "RG20 7JN",
        "RG20 7JP",
        "RG20 7JQ",
        "RG20 7JR",
        "RG20 7JS",
        "RG20 7JT",
        "RG20 7JW",
        "RG20 7JX",
        "RG20 7JY",
        "RG20 7JZ",
        "RG20 7LA",
        "RG20 7LB",
        "RG20 7LD",
        "RG20 7LE",
        "RG20 7LF",
        "RG20 7LG",
        "RG20 7LH",
        "RG20 7LJ",
        "RG20 7LL",
        "RG20 7LN",
        "RG20 7LP",
        "RG20 7LQ",
        "RG20 7LR",
        "RG20 7LS",
        "RG20 7LT",
        "RG20 7LU",
        "RG20 7LW",
        "RG20 7LX",
        "RG20 7LY",
        "RG20 7LZ",
        "RG20 7NA",
        "RG20 7ND",
        "RG20 8BD",
        "RG20 8QG",
        "RG20 8QT",
        "RG20 8QU",
        "RG20 8QX",
        "RG20 8RA",
        "RG20 8RB",
        "RG20 8RD",
        "RG20 8RE",
        "RG20 8RG",
        "RG20 8RH",
        "RG20 8RJ",
        "RG20 8RN",
        "RG20 8TA",
        "RG20 8TB",
        "RG20 8TD",
        "RG20 8TE",
        "RG20 8TF",
        "RG20 8TG",
        "RG20 8TH",
        "RG20 8TJ",
        "RG20 8TL",
        "RG20 8TN",
        "RG20 8TP",
        "RG20 8TR",
        "RG20 8TS",
        "RG20 8TU",
        "RG20 8TW",
        "RG20 8TY",
        "RG20 8TZ",
        "RG20 8UA",
        "RG20 8UB",
        "RG20 8UD",
        "RG20 8UE",
        "RG20 8UG",
        "RG20 8UJ",
        "RG20 8UR",
        "RG20 8XB",
        "RG20 8XD",
        "RG20 8XF",
        "RG20 8XG",
        "RG20 8XQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4002,
              51.5625
            ],
            [
              -1.2455,
              51.5625
            ],
            [
              -1.2455,
              51.398
            ],
            [
              -1.4002,
              51.398
            ],
            [
              -1.4002,
              51.5625
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5625,
      "etrUpdate": true
    },
    {
      "latitude": 51.52823301859986,
      "type": "HV",
      "reference": "MW6722",
      "etaUpdate": true,
      "affectedCustomerCount": 312,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7d18a05e-88ae-4e91-8285-fbff19960042",
      "minLongitude": -1.3304,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.293378984991907,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T02:33:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6722'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6722",
      "maxLongitude": -1.2605,
      "minLatitude": 51.5116,
      "name": "RG20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX11 0RP",
        "RG20 7AD",
        "RG20 7AE",
        "RG20 7AG",
        "RG20 7AJ",
        "RG20 7AW",
        "RG20 7DJ",
        "RG20 7HZ",
        "RG20 7JB",
        "RG20 7JZ",
        "RG20 7LA",
        "RG20 7LB",
        "RG20 7LD",
        "RG20 7LE",
        "RG20 7LF",
        "RG20 7LG",
        "RG20 7LH",
        "RG20 7LJ",
        "RG20 7LL",
        "RG20 7LN",
        "RG20 7LP",
        "RG20 7LQ",
        "RG20 7LR",
        "RG20 7LS",
        "RG20 7LT",
        "RG20 7LU",
        "RG20 7LW",
        "RG20 7LX",
        "RG20 7LY",
        "RG20 7LZ",
        "RG20 7NA",
        "RG20 7ND"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3304,
              51.5625
            ],
            [
              -1.2605,
              51.5625
            ],
            [
              -1.2605,
              51.5116
            ],
            [
              -1.3304,
              51.5116
            ],
            [
              -1.3304,
              51.5625
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5625,
      "etrUpdate": true
    },
    {
      "latitude": 50.85119086168846,
      "type": "LV",
      "reference": "MW6719",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "dcc54af2-8ce2-4015-92ad-a7bc19dd5ce2",
      "minLongitude": -1.4115,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4097840290095476,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T02:29:00.000+0000",
      "estimatedRestoration": "2023-11-26T05:35:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6719'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6719",
      "maxLongitude": -1.4075,
      "minLatitude": 50.8493,
      "name": "SO45 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO45 4JY",
        "SO45 4JZ",
        "SO45 4LA",
        "SO45 4LB",
        "SO45 4LD",
        "SO45 4LE",
        "SO45 4LF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4115,
              50.8525
            ],
            [
              -1.4075,
              50.8525
            ],
            [
              -1.4075,
              50.8493
            ],
            [
              -1.4115,
              50.8493
            ],
            [
              -1.4115,
              50.8525
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8525,
      "etrUpdate": true
    },
    {
      "latitude": 51.44071250966889,
      "type": "LV",
      "reference": "MW6724",
      "etaUpdate": true,
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "15f0cca4-cae4-4f98-8a6a-b3dbacaa63f5",
      "minLongitude": -1.3218,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2985570559791753,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-26T02:40:00.000+0000",
      "estimatedRestoration": "2023-11-26T05:40:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MW6724'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MW6724",
      "maxLongitude": -1.1957,
      "minLatitude": 51.319,
      "name": "RG20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG20 8TE",
        "RG20 8UP",
        "RG20 8UR",
        "RG20 8UT",
        "RG20 8UX",
        "RG26 5RU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3218,
              51.4675
            ],
            [
              -1.1957,
              51.4675
            ],
            [
              -1.1957,
              51.319
            ],
            [
              -1.3218,
              51.319
            ],
            [
              -1.3218,
              51.4675
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4675,
      "etrUpdate": true
    },
    {
      "latitude": 57.072828932689674,
      "type": "PSI",
      "reference": "221043",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "eeceb619-98c2-4597-8534-a4383bc97d87",
      "minLongitude": -4.1316,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.0765997166515335,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-25T22:00:00.000+0000",
      "estimatedRestoration": "2023-11-26T08:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '221043'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/221043",
      "maxLongitude": -4.0357,
      "minLatitude": 57.0605,
      "name": "PH20 & PH21 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "PH20 1EF",
        "PH21 1ES",
        "PH21 1HW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.1316,
              57.0801
            ],
            [
              -4.0357,
              57.0801
            ],
            [
              -4.0357,
              57.0605
            ],
            [
              -4.1316,
              57.0605
            ],
            [
              -4.1316,
              57.0801
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.0801,
      "etrUpdate": true
    }
  ]
}
